### ISSUE TITLE
Verify case study results are reproducible with final DISPATCHES release candidate

### ISSUE THREAD

**Issue by adowling2** [gmlc-dispatches/dispatches#207]
Goal: verify the key results in each case study report are reproducible with the final DISPATCHES release candidate

RE case study (generic):
- [ ] @dguittet price taker for RE case study
- [ ] @Xinhe-Chen train surrogates for RE case study*
- [ ] @Xinhe-Chen solve surrogates embedded optimization for RE case study*
- [ ] @dguittet @Xinhe-Chen Prescient verification simulation

NE case study (generic):
- [ ] @radhakrishnatg price-taker for NE case study
- [ ] @Xinhe-Chen train surrogates for NE case study*
- [ ] @Xinhe-Chen @radhakrishnatg solve surrogates embedded optimization for NE case study*
- [ ] @radhakrishnatg @Xinhe-Chen Prescient verification simulations

*TensorFlow will likely be pinned to 2.12 (or 2.10)

FE case study:
- [ ] @esrawli GDP optimization
- [ ] @nareshsusarla multiperiod optimization

Others:
- [ ] @dguittet RE collaborative case study
- [ ] @radhakrishnatg NE collaborative case study

Instructions:
- Post to this issue details about which files you ran for verification. We will use comments on this issue as documentation.
- Once the verification is done, check off the task above.

**Comment by adowling2** [comment]
@dguittet @radhakrishnatg @esrawli @nareshsusarla @Xinhe-Chen To recap, our goal is to have the scripts/notebooks used to generate key results from the reports included in the final release. If the scripts/notebooks are not in the release candidate, please work with @lbianchi-lbl to open and merge a PR.

**Comment by adowling2** [comment]
@lbianchi-lbl I am following these instructions on an M1 macOS (with a recent branch on @nareshsusarla's fork) and got this error:
```
INFO: pip is looking at multiple versions of dispatches[surrogates] to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement tensorflow==2.10.0 (from dispatches[surrogates]) (from versions: 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0)
ERROR: No matching distribution found for tensorflow==2.10.0
```

**Comment by adowling2** [comment]
I decided to do ``pip install --editable .`` because I want to see if some scripts in @nareshsusarla's fork run on my computer (and `tensorflow` is not needed), so this is unimportant.

**Comment by lbianchi-lbl** [comment]
> @lbianchi-lbl I am following these instructions on an M1 macOS (with a recent branch on @nareshsusarla's fork) and got this error:
> 
> ```
> INFO: pip is looking at multiple versions of dispatches[surrogates] to determine which version is compatible with other requirements. This could take a while.
> ERROR: Could not find a version that satisfies the requirement tensorflow==2.10.0 (from dispatches[surrogates]) (from versions: 2.13.0rc0, 2.13.0rc1, 2.13.0rc2, 2.13.0)
> ERROR: No matching distribution found for tensorflow==2.10.0
> ```

Is this in a fresh Conda environment? If so, the Python version could be an issue - for various reasons DISPATCHES is only compatible with 3.8 or 3.9, with the latter being the one used for the instructions above.

Running `pip install --editable .` is a good alternative to `pip install -r requirements-dev.txt`, modulo some possible test failures due to e.g. `addheader`, but if things work, then no need to change.

**Comment by adowling2** [comment]
Yes, a new conda environment with Python 3.9. I wonder if that version of tensorflow is not available for macOS arm.

**Comment by lbianchi-lbl** [comment]
> Yes, a new conda environment with Python 3.9. I wonder if that version of tensorflow is not available for macOS arm.

Oh, right, sorry, for some reason I skipped the part about the M1 Mac part in your message. Yes, I'd say it's likely, especially considering that 2.10 is relatively old. From a quick check, it looks like the `arm64` build is only available (on PyPI) since Tensorflow 2.13: https://pypi.org/project/tensorflow/2.13.0/#files
